.buildkite/scripts/pipelines/security_solution_quality_gate/api_integration/api-integration-tests.sh

```diff
@@ -18,12 +18,29 @@ QA_API_KEY=$(vault_get security-solution-quality-gate qa_api_key)
 
 # Generate a random 5-digit number
 random_number=$((10000 + $RANDOM % 90000))
-ENVIRONMENT_DETAILS=$(curl --location 'https://global.qa.cld.elstc.co/api/v1/serverless/projects/security' \
+if [ -z "${KIBANA_MKI_USE_LATEST_COMMIT+x}" ] || [ "$KIBANA_MKI_USE_LATEST_COMMIT" = "0" ]; then
+  ENVIRONMENT_DETAILS=$(curl --location 'https://global.qa.cld.elstc.co/api/v1/serverless/projects/security' \
```

Comment:
Is this a private URL? if so, we should not hardcoded here.

Reply:
https://github.com/elastic/kibana/pull/171859/commits/938435d662971e0832c68b6a810fff26f2a4033a
Done